guilder-syncer.yaml

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
This file was an unintentional inclusion, right?

@clubanderson , I thought we had something that screens for secrets before pushing to github?

Reply:
They were indeed unintentional inclusions...artifacts from testing